cli/parsefile.go

```diff
@@ -200,3 +207,36 @@ func extractPmt(buf io.Reader, pid uint16) (psi.PMT, error) {
 	}
 	return pmt, nil
 }
+func verifySync(buf io.Reader) (bool,int64){
+   //function find the first sync byte of the array   
+  	pkt := make([]byte, 1)
+    var i int64 = 0
+	for {
```

Comment:
This can be formatted as a standard for-loop construct which makes the code read a bit more clearly: 
`for i := int64(0); ;i++ {`